std_msgs/Header header
aruco_opencv/MarkerPose[] markers
